amf-api-contract/shared/src/main/scala/amf/apicontract/client/scala/AMFConfiguration.scala

```diff
@@ -284,6 +285,9 @@ object APIConfiguration extends APIConfigurationBuilder {
 
   private val name = "API"
 
+  def Schemas(): AMFConfiguration = API()
```

Comment:
This is a weird name because Schemas is not narrow enough. Probably break with the 'spec name' pattern and be able to enrich any configuration?

Reply:
What name do you propose?

Reply:
`JsonSchema` ? jaja Im uneasy about the default being the API() though. I don't think this pattern will create a clear vision about API() being the default though

Reply:
I thought on JsonSchema at the beginning, but I think could close the option to add "other" thinks in a future (another king of schema? not sure).
Maybe will never have another kind of schema based on syaml to support, but add JsonSchema sound  ver restrictive (and not sure if clear that is capable to parse another specs as well)